pkgs/applications/misc/robo3t/default.nix

```diff
@@ -43,6 +43,10 @@ stdenv.mkDerivation rec {
     fontconfig
     freetype
     libGL
+    (curl.override {
+      gnutlsSupport = true;
+      sslSupport = false;
+    })
```

Comment:
Please move this into top level and override a curl that is inherited.

Reply:
Moved it top level (as in other packages using curl with gnutls). Not 100% sure if that's what you meant though ;)